src/cli_parser.cpp

```diff
@@ -89,6 +89,14 @@ void CLIParser::parse(int argc, char** argv) {
                 "Overrides model cache directory. By default cache files are saved into /opt/cache if the directory is present. When enabled, first model load will produce cache files.",
                 cxxopts::value<std::string>(),
                 "CACHE_DIR")
+            ("metrics_enable",
+                "Flag enabling metrics endpoint on rest_port.",
+                cxxopts::value<bool>()->default_value("false"),
+                "METRICS")
+            ("metrics_list",
```

Comment:
Should we include information that when used, config.json cannot contain metric configuration? @dtrawins

Reply:
I would keep it short but indeed documentation should be adjusted in https://github.com/openvinotoolkit/model_server/blob/develop/docs/parameters.md 
A note in https://github.com/openvinotoolkit/model_server/blob/main/docs/metrics.md could be also useful that configuration can not be duplicated.